src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -341,5 +345,20 @@ class LazyVals extends MiniPhaseTransform with SymTransformer {
     }
 }
 
+object LazyVals {
+  val RLazyValsNames_get               = RLazyVals.Names.get.toTermName
+  val RLazyValsNames_setFlag           = RLazyVals.Names.setFlag.toTermName
+  val RLazyValsNames_wait4Notification = RLazyVals.Names.wait4Notification.toTermName
+  val RLazyValsNames_state             = RLazyVals.Names.state.toTermName
+  val RLazyValsNames_cas               = RLazyVals.Names.cas.toTermName
+  val RLazyValsNames_getOffset         = RLazyVals.Names.getOffset.toTermName
+  val nme_flag                         = "flag".toTermName
+  val nme_result                       = "result".toTermName
+  val nme_value                        = "value".toTermName
+  val nme_initialized                  = "initialized".toTermName
+  val nme_retry                        = "retry".toTermName
+
+}
+
```

Comment:
I think naming is a bit of a mess here. It would be more consistent to spend another object, say `lazyNme` and collect all names in it. Something like:

    object LazyVals {
      object lazyNme {
        val get = RLazyVals.Names.get.toTermName
        ...
        val flag = "flag".toTermName
        ...
      }
    }